project/query.py

```diff
@@ -0,0 +1,32 @@
+"""Query class."""
+
+import os
+import tweepy
+
+class Query(object): # pylint: disable=too-few-public-methods
+    """Class to query multiple API's."""
+    def __init__(self):
+        # Authenticates multiple API's
+
+        # Twitter authentification process
+        twit_consumer_key = os.environ.get('TWITTER_CONSUMER_KEY')
+        twit_consumer_secret = os.environ.get('TWITTER_CONSUMER_SECRET')
+        twit_access_token = os.environ.get('TWITTER_ACCESS_TOKEN')
+        twit_access_token_secret = os.environ.get('TWITTER_ACCESS_TOKEN_SECRET')
+        twitter_auth = tweepy.OAuthHandler(twit_consumer_key,
+                                           twit_consumer_secret)
+        twitter_auth.set_access_token(twit_access_token,
+                                      twit_access_token_secret)
+        self.twitter_api = tweepy.API(twitter_auth)
+
+
+    def twitter_search(self, query):
```

Comment:
Are we having the caller call twitter_search and news_search, etc? Or will we have a single search function that calls the search functions for each API/scraper?